airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/read/JdbcPartitionReader.kt

```diff
@@ -37,8 +36,8 @@ sealed class JdbcPartitionReader<P : JdbcPartition<*>>(
         return PartitionReader.TryAcquireResourcesStatus.READY_TO_RUN
     }
 
-    fun out(record: ObjectNode, changes: Map<Field, FieldValueChange>?) {
-        streamRecordConsumer.accept(record, changes)
+    fun out(row: SelectQuerier.ResultRow) {
```

Comment:
this `ResultRow` thing is basically a pair of `ObjectNode` and `Map<Field, FieldValueChange>`